packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -11,7 +11,8 @@
   "credential_verify_request_right_pane_top_title": "Provide details",
   "credential_verify_request_right_pane_top_paragraph": "In order to issue a credential we need to get\nsome information from your wallet.",
   "button_open_from_wallet": "Open wallet",
-  "credential_verify_request_right_pane_bottom_paragraph": "Scan this QR code using your wallet's QR\nscanner to provide information.",
+  "credential_verify_request_right_pane_bottom_title": "Sign In",
```

Comment:
Don't change existing language resources from other demos, create a new one. (then also for Dutch)

Reply:
Fixed!